packages/tldraw/src/lib/ui/version.ts

```diff
@@ -2,3 +2,8 @@
 // Do not edit manually. Or do, I'm a comment, not a cop.
 
 export const version = '2.2.0'
+export const publishDates = {
+	major: '2024-06-28T10:56:07.893Z',
```

Comment:
should these be GMT? i.e. '2024-06-28T00:00:00.000Z'
i'm just wondering about the day offset problem we were talking about

Reply:
Handled that by throwing away the hour / timezone part of the date and added an additional day to that.
https://github.com/tldraw/tldraw/pull/4036/files#diff-6e7aadaa37ccd9daf67f2589ea799f33bf1c67b5a47ace64d4096ee40f460358R100